aten/src/ATen/native/native_functions.yaml

```diff
@@ -681,15 +681,29 @@
   structured_delegate: all.out
   variants: function, method
 
+- func: all.dims(Tensor self, int[1]? dim=None, bool keepdim=False) -> Tensor
```

Comment:
Question: Does `int[1]` imply the length of the sequence can only be 1? I have seen the use of `int[]` too. Was this intended?

Reply:
`int[1]` is a broadcasting list, it means if you pass a scalar it will be broadcast to a list of size 1. I'm removing the broadcasting aspect though as it's covered by the `all.dim` overload (I had hoped  we wouldn't need two).